roles/openshift_node/meta/main.yml

```diff
@@ -15,3 +15,8 @@ dependencies:
 - role: openshift_docker
 - role: openshift_cloud_provider
 - role: openshift_common
+- role: openshift_node_dnsmasq
+  dns_domain: "{{ openshift.common.dns_domain }}"
+  kube_svc_ip: "{{ openshift.common.kube_svc_ip }}"
+  when: openshift_node_dnsmasq | default(True if openshift.common.version_gte_3_2_or_1_2 | bool else False) | bool
```

Comment:
There should be a fact value added for `openshift.node.config_dnsmasq` or similar and have the value defaulted in openshift_facts based on the version number rather than computing it here.

This will require pushing the node facts into an openshift_node_facts role, but @abutcher has a PR that could be used as a guide.